### ISSUE TITLE
Gamepad settings

### ISSUE THREAD

**Issue by LAWG44** [Vita3K/Vita3K-Android#515]
Only buttons 1 to 10 can be configured on a gamepad, buttons such as L2 or R2 cannot be used to play games

**Comment by Professor77** [comment]
PS4 L2/R2 inputs are not detected when rebinding controller inputs. Would it be possible to add L2/R2 to the controller bindings for use with PSTV mode?

**Comment by LAWG44** [comment]
I have activated pstv mode, but there are no changes

**Comment by Professor77** [comment]
Yes, the L2/R2 buttons are added to touchscreen only, they don't match up with the gamepad triggers.

**Comment by LAWG44** [comment]
Exactly

**Comment by zgillet** [comment]
This is quite annoying for shooter games. Currently playing EDF 2, and I'd love to switch the triggers and shoulder buttons.
